src/Card/CardSection/index.js

```diff
@@ -3,27 +3,144 @@ import * as React from "react";
 import styled from "styled-components";
 
 import defaultTokens from "../../defaultTokens";
+import ChevronDown from "../../icons/ChevronDown";
+import Heading from "../../Heading/index";
+import Text from "../../Text/index";
+import { getSize } from "../../Icon";
+import { ICON_SIZES } from "../../Icon/consts";
 
 import type { Props } from "./index";
 
+const StyledCardSectionIconRight = styled(ChevronDown)`
+  position: absolute;
+  right: 0;
+  top: 50%;
+  margin-top: -12px;
```

Comment:
Can you plz use flex here? We will not have to define tokens for this, or parseInt it.